src/assets/scripts/client/base/ModelSource/modelSourceClassMap.js

```diff
@@ -17,7 +18,15 @@ export const CLASS_MAP = {
      * @type {FixModel}
      * @final
      */
-    FixModel: FixModel
+    FixModel: FixModel,
```

Comment:
pretty close.  the `CLASS_MAP` give us a map of classes that can be used in the modelSourcePool.  These classes will have `n` number of instances pre-made when the app loads and are then used throughout the app when needed and returned to the pool when not-needed.

these guys get used thusly:
```javascript
    releaseReusable(constructorName, ...args) {
        let model = this._findModelByConstructorName(constructorName);

        if (!model) {
            model = new CLASS_MAP[constructorName](...args);
        }

        model.init(...args);

        return model;
    }

// and the called
// .getModelSourceForType() is a facade for .releaseReusable()
            const spawnPatternModel = modelSourceFactory.getModelSourceForType(
                // constructorName
                'SpawnPatternModel',
                // ...args (everything else)
                spawnPattern,
                navigationLibrary
            );
```